vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1094,6 +1095,10 @@ std::vector<std::unique_ptr<ExclusionRule>>
       std::make_unique<TotalMaxFrequencyCap>(frequency_capping_.get());
   exclusion_rules.push_back(std::move(total_max_frequency_cap));
 
+  std::unique_ptr<ExclusionRule> conversion_frequency_cap =
```

Comment:
nit: could be auto

Reply:
I only use `auto` now for complex types as makes code harder to read